easybuild/easyconfigs/g/GSL/GSL-2.3-foss-2016b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '2.3'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+configopts = '--with-pic'
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'include', 'lib', 'share'],
```

Comment:
@verdurin I'd go with something more specific, like:

```python
sanity_check_paths = {
    'files': ['bin/gsl-config', 'lib/libgsl.a', 'lib/libgsl.%s' % SHLIB_EXT,
              'lib/libgslcblas.a', 'lib/libgslcblas.%s' % SHLIB_EXT],
    'dirs': ['include/gsl', 'lib/pkgconfig', 'share'],
}
```

done in https://github.com/verdurin/easybuild-easyconfigs/pull/5